### PR TITLE
Welcome page

### PR DESCRIPTION
**Pull request by whiteNsky**
#Welcome Page

### UPDATE V1.0.0

- [x] Added clickable cursor
- [x] Linked the welcome page to login page

### CODE REVIEW

**Comment by Techies-John** [comment]
Tarunga ang PR and commit message PJ sunda ang correct format

**Comment by Techies-John** [comment]
Tabangi kog butang og tango sa atong balaod para ma organize ta tanan

File: components/WelcomePage.vue
Comment (Techies-John) [inline components/WelcomePage.vue:1]: Sunda ni nga file directory and naming: 
components->Welcome->Page.vue

**Review by Techies-John — CHANGES_REQUESTED** [review]
Overall chada kaayo syag design but,

- Avoid having two words in pages as much as possible. If "homepage" then "home" ra, if "aboutpage" then "about" ra.

- Also, try separate components by adding folders base on pages.

**Review by Techies-John — APPROVED** [review]

**Review by sggenDebugs — APPROVED** [review]

**Pull request merged**